examples/minimal.rs

```diff
@@ -202,21 +227,24 @@ async fn start_node(
     let contact_info = node
         .our_connection_info()
         .expect("Failed to obtain node's contact info.");
+    let handler: JoinHandle<()> = run_node(index, node, event_stream).await;
 
-    run_node(index, node, event_stream).await;
-
-    contact_info
+    (contact_info, handler)
 }
 
 // Runs the nodes event loop. Blocks until terminated.
```

Comment:
This comment is outdated. Maybe change to something like "spawns a task to run the node until terminated".